client/src/App.js

```diff
@@ -3,15 +3,14 @@ import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import { ToastContainer } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
 
-
 import Login from "./pages/auth/Login";
 import Register from "./pages/auth/Register";
 import Navbar from "./pages/layout/Navbar";
 import HomePage from "./pages/home/HomePage";
 import Dashboard from "./pages/user/Dashboard";
+import AdminDashboard from "./pages/admin/AdminDashboard";
 import PrivateRoute from "./pages/routing/PrivateRoute";
-import About from './pages/about/About';
-
+import About from "./pages/about/About";
```

Comment:
you have to align with @AlsasaAmmar with the format. You can use double or single quotes. It is not really important but it is important to align the code styles between you. I would recommend you to add eslint and prettier to the project, you can see here some useful links:

https://thomlom.dev/setup-eslint-prettier-react/